src/test/java/de/tum/in/www1/artemis/TextSubmissionIntegrationTest.java

```diff
@@ -91,4 +95,18 @@ public void getResultsForCurrentStudent_assessorHiddenForStudent() throws Except
         assertThat(returnedExercise.getParticipations().iterator().next().getResults().iterator().next().getAssessor()).as("assessor is null").isNull();
     }
 
+    @Test
+    @WithMockUser(value = "student1", roles = "USER")
+    public void getDataForTextEditorWithResult() throws Exception {
+        TextSubmission textSubmission = database.addTextSubmissionWithResultAndAssessor(textExercise, this.textSubmission, "student1", "tutor1");
+        Long participationId = textSubmission.getParticipation().getId();
+
+        StudentParticipation participation = request.get("/api/text-editor/" + participationId, HttpStatus.OK, StudentParticipation.class);
+
+        Assert.assertThat(participation.getResults(), is(notNullValue()));
```

Comment:
I personally like to use `import static org.assertj.core.api.Assertions.assertThat;`.
Then it looks like this: `assertThat(reflog.size()).isEqualTo(1);`

Reply:
not sure how that could slip through.